main.js

```diff
@@ -50,18 +50,20 @@ savedPalettesSection.addEventListener('click', async function(event) {
     }
 });
 
-function getPromiseFromEvent(element, listenerName) {
+function getPromiseFromEvent(event) {
+
     return new Promise(function (resolve) {
+        event.addEventListener('click', listener);
+
```

Comment:
Another empty line, is this on purpose to make it clear there's another function being created? If so, I get it.

Reply:
Yeah, the purpose was to make it clear, not sure if I saw it on the styling doc or if one of the instructors said something but I think we should have an empty line here. I removed the one on line 53.